README.md

```diff
@@ -56,11 +69,47 @@ In this order:
   ```sh
   git clone git@github.com:wc-phdc-hia/testing.git
   ```
+
+- To get the latest Playwright run:
+  
+  ```sh
+  npm install -D @playwright/test@latest
+  ```
+
+- A package for setting environment variables:
+  
+  ```sh
+  npm install -D env-cmd
+  ```
   
+>**Important:** Create a .env file in the root folder that will contain your local SPV username and password. i.e. USERNAME="username here", PASSWORD="password here"
+
 ## Resources
 
 [Playwright](https://playwright.dev/docs/intro)
 
+### Regression test
+
+Prerequisite
+
+- All values in .env file from should not be empty.
```

Comment:
What are the values that need to be populated? This could really do with an `.env.example` file.

Reply:
done